.travis.yml

```diff
@@ -25,7 +24,9 @@ matrix:
     fast_finish: true
     include:
         - php: 5.4
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci"
+          env: COMPOSER_FLAGS="--ignore-platform-reqs"
```

Comment:
No, it is required for php-mock, which I used to mock `extension_loaded` function to test exceptions thrown.